docker-compose.yml

```diff
@@ -98,7 +98,11 @@ services:
             - db-data:/var/lib/postgresql/data
 
     headless-browser:
-        image: ghcr.io/browserless/chromium:v2.24.3
+        build:
+            context: .
+            dockerfile: docker/Dockerfile.headless-browser
```

Comment:
Do we need to update helm-chart?

Reply:
Yeah! I checked everything, and we have a few options: we can disable some of the log levels, do this or contribute to their repo.

helm chart needs to be updated, yes, but I’ll test it more thoroughly before making any changes 👌